gpiozero/output_devices.py

```diff
@@ -503,6 +504,53 @@ def _blink_device(
             self._write(value)
             if self._blink_thread.stopping.wait(delay):
                 break
+            
+class PWMBuzzer(PWMOutputDevice):
+    """
+    Extends :class: `PWMOutputDevice` and represents a piezo or tonal buzzer 
+    class.
+
+    :param: int pin:
+        The GPIO pin which the buzzer is attached to. See :ref:`pin-numbering` for
+        valid pin numbers.
+    
+    :param bool active_high:
+        If ``True`` (the default), the :meth:`on` method will set the GPIO to 
```

Comment:
Hmmm, if the buzzer functions by means of pwm, do we even need to specify whether it's active-high or active-low?

Reply:
Good point. We can get rid of that option.